src/cpl/nuopc/rof_import_export.F90

```diff
@@ -113,10 +113,8 @@ subroutine advertise_fields(gcomp, flds_scalar_name, rc)
     call fldlist_add(fldsToRof_num, fldsToRof, 'Flrl_rofsub')
     call fldlist_add(fldsToRof_num, fldsToRof, 'Flrl_rofi')
     call fldlist_add(fldsToRof_num, fldsToRof, 'Flrl_irrig')
-    if ( ctl%rof_from_glc ) then
```

Comment:
This was the commit that fixed an important bug in the coupled model. Coupled simulations done with cesm3_0_beta05 had a problem and then fixed in cesm3_0_beta06.

This fixed #103